Adding steps for how to use existing cluster as bootstrap cluster. (#877)
kubernetes-sigs/cluster-api: cmd/clusterctl/README.md

### Prerequisites

> 1. You can use either VM, container or existing Kubernetes cluster act as bootstrap cluster, it is recommended to use container or existing Kubernetes cluster.

Reviewer: ```suggestion
1. Cluster API runs its operations in Kubernetes. A pre-existing or temporary bootstrap cluster is required. Currently, we support multiple methods to bootstrap Cluster API: `kind` (preferred), `minikube` or any pre-existing cluster.
```

   - If you want to use VM, install [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), version 0.30.0 or greater.

>    - If you want to use container, install [kind](https://github.com/kubernetes-sigs/kind#installation-and-usage).

Author: Done

1. Create a cluster:

   - With `bootstrap-type`. Currently two `bootstrap-type` options are supported - `kind` and `minikube`.

   ```shell
   ./clusterctl create cluster --provider <provider> --bootstrap-type <bootstrap-type> -c cluster.yaml \
     -m machines.yaml -p provider-components.yaml -a addons.yaml
   ```

   If you are using minikube, to choose a specific minikube driver, please use the `--bootstrap-flags vm-driver=xxx` command line parameter. For example to use the kvm2 driver with clusterctl you woud add `--bootstrap-flags vm-driver=kvm2`.

>    - With `bootstrap-cluster-kubeconfig`, this is used when you are using an existing Kubernetes cluster as your bootstrap cluster.

Reviewer: ```suggestion
   -  __Existing Cluster__:  Use `bootstrap-cluster-kubeconfig`. This flag is used when you have an existing Kubernetes cluster.
```
